public/docs/ganache/using.md

```diff
@@ -0,0 +1,64 @@
+# Working with Ganache
+
+[Ganache](/ganache) is an application that generates a personal blockchain you can use to run tests, execute commands, deploy contracts, and inspect state.
+
+![Ganache logo](/docs/ganache/images/ganache-logo.png)
+
+It is available in both a graphical format and as a terminal application (which was previously known as the TestRPC). Ganache is available for Windows, macOS, and Linux.
+
+## Installation
+
+### Ganache
+
+Installation of Ganache is only a few steps. [Download](https://github.com/trufflesuite/ganache/releases) the appropriate version for your OS:
+
+* Windows: `Ganache-a.b.c.appx`
+* macOS: `Ganache-a.b.c.dmg`
+* Linux: `Ganache-a.b.c.AppImage`
+
+Double click on the downloaded file, follow the prompts, and you're up and running.
+
+### Ganache CLI
+
+If you are interested in only the command-line application (previously known as the TestRPC), you can get it through npm.
+
+```shell
+npm install -g ganache-cli
+```
+
+This page will focus only on the graphical interface. Please see the [README](https://github.com/trufflesuite/ganache-cli/blob/master/README.md) for more information on Ganache CLI.
+
+## Main interface
+
+When you first launch Ganache, the screen will show some details about the server, and also list out a number of accounts. You'll note that each account has been given 100 ether, to make testing of transactions much easier.
```

Comment:
This is actually not true. On your _very first_ run, you're asked whether or not you want to allow Google analytics tracking. Should we add that? 

Switch "testing of transactions" with "developing your application" easier (i.e., focus on development in general instead of testing specifically)

Reply:
Oh good point. I'll add a note about Google Analytics.